opennms-container/minion/container-fs/confd/templates/prom-jmx-exporter.yaml.tmpl

```diff
@@ -1,6 +1,8 @@
 #
```

Comment:
You'll need to include the comments inside the if condition otherwise the file will not be empty and won't be removed.